homework.md

```diff
@@ -4,110 +4,112 @@
 
 What is 2 + 2?
 
-5
+Linus, 2+2 is 4, not 5, PUT ATTENTION STOP PLAYING POKEMON GO ON CLASSES!
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White, Linus are you daltonic?
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12240, I need to accept I didn't even knew this.
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hypertext Markup Language (HTML) (I didn't even knew this)
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Making a website looks pretty, the hype for graphic designers and editors.
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
 ```css
-#h1 {
+<h1> { Here is the section for heading elements
 
-}
+} </h1>
 ```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+Javascript is a language that gives interface to the web pages adding interactive elements that catch the attention of the user. 
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+Github is a community of developers where ideas and works can be shared.
+Makes easy the collaborators between career colleagues.
+Team members can work on files and they can easily merge changes.
```

Comment:
Cool :clap: 
Git was created to solve some important issues that developers faced in the past. This was covered in the first lesson of the Git week. Can we remember that those things are? Tip: you can always revisit the material ;)
https://syllabus.codeyourfuture.io/git/index